### ISSUE TITLE
Onfido framework conflicts with Segment Analytics

### ISSUE THREAD

**Issue by howelse** [onfido/onfido-ios-sdk#55]
It looks like the Segment Analytics sources are compiled into the Onfido framework, which leads to compatibility issues at runtime.

### Version info:

- Cocoapods (if applicable): 1.5.3
- Onfido SDK (Debug/Release): 10.1.0
- iOS: 12.1
- Xcode: 10.1
- Device/Simulator: iPhone X
- Device/Simulator language:

### Integration configuration:

Podfile
```
target 'SampleApp' do
  use_frameworks!

  pod 'Onfido'
  pod 'Alamofire'
  pod 'Analytics'
end
```

### Steps to reproduce:

The following warning appears in console for each Segment Analytics class
```
objc[1161]: Class SEGAnalytics is implemented in both /Users/howelse/Library/Developer/CoreSimulator/Devices/3582247D-F6FE-4A64-87CD-02714CB5F101/data/Containers/Bundle/Application/1611A03A-C8A9-4FA0-9441-2DC161AB59F8/SampleApp.app/Frameworks/Onfido.framework/Onfido (0x105f764e0) and /Users/howelse/Library/Developer/CoreSimulator/Devices/3582247D-F6FE-4A64-87CD-02714CB5F101/data/Containers/Bundle/Application/1611A03A-C8A9-4FA0-9441-2DC161AB59F8/SampleApp.app/Frameworks/Analytics.framework/Analytics (0x105219a70). One of the two will be used. Which one is undefined.
objc[1161]: Class SEGAnalyticsConfiguration is implemented in both /Users/howelse/Library/Developer/CoreSimulator/Devices/3582247D-F6FE-4A64-87CD-02714CB5F101/data/Containers/Bundle/Application/1611A03A-C8A9-4FA0-9441-2DC161AB59F8/SampleApp.app/Frameworks/Onfido.framework/Onfido (0x105f76530) and /Users/howelse/Library/Developer/CoreSimulator/Devices/3582247D-F6FE-4A64-87CD-02714CB5F101/data/Containers/Bundle/Application/1611A03A-C8A9-4FA0-9441-2DC161AB59F8/SampleApp.app/Frameworks/Analytics.framework/Analytics (0x105219ac0). One of the two will be used. Which one is undefined.
```

**Comment by anuragajwani** [comment]
Hi @howelse,

Thanks for raising the issue. We are looking into solving this issue for our next release.

Sorry for the inconvenience,
Anurag

**Comment by anuragajwani** [comment]
Hi @howelse,

The fix is now being reviewed.

Anurag

**Comment by AndreasVerhoeven** [comment]
@anuragajwani is there any progress on this issue? We're also running into this problem. iOS/Mac frameworks generally should prefix all of their symbols (even external ones) to avoid these types of issues.

**Comment by anuragajwani** [comment]
Hi @AndreasVerhoeven @howelse  this issue has been fixed on Onfido SDK 10.2.0 https://github.com/onfido/onfido-ios-sdk/releases/tag/10.2.0

**Comment by AndreasVerhoeven** [comment]
This seems solved for us! Thanks!
